musicbot/bot.py

```diff
@@ -226,6 +223,21 @@ def _setup_logging(self):
             dhandler.setFormatter(logging.Formatter('{asctime}:{levelname}:{name}: {message}', style='{'))
             dlogger.addHandler(dhandler)
 
+# Here's Where log clean up comes in
+    now = time.time()
+    cutoff = now - (1 * 86400)
```

Comment:
I was working within my current skills set with python multiplying by 1 and 86400 will always result in the number coming out as 86400 which in seconds is a day entering the day in seconds did not work for me it had to be a math problem so to make old logs delete after a day I just had it multiply 1 * 86400 so it would always delete the old logs after a day

Reply:
by simply putting 86400 it would not work correctly so I made it a multiplication problem that would result in the logs being deleted after a day

Reply:
now = time.time() alls this code to see what time it is if it sees the a file's age exceeds 86400 seconds it would delete thoses log files so it cleans up everything this code is run the way I did it however means the bot needs to be restarted in order to delete the old logs

Reply:
I've also noticed a side effect of running this on Windows it deletes logs based on their creation date so depending on how many times someone restarts their bot in a day it will result in them either having a lot of logs or little it depends on how many times the bot is restarted a day

Reply:
also I've tested the code very thoroughly it does work on python 3.5 and 3,6 and I agree with  jaydenkieran the system you guys have right now isn't suitable enough to keep around so this code does create logs based on time and date and in order to start the clean up someone has to restart their bot